### ISSUE TITLE
Dart hotfixes blocked on beta/stable: Flutter engine depends on unstable Windows arm64 Dart SDK

### ISSUE THREAD

**Issue by athomas** [flutter/flutter#113931]
The Flutter Engine depends on the unstable Dart SDK for Windows ARM64:
https://github.com/flutter/engine/blob/6bb2f03e6fd00b440e6ab2e20ad94b8104cc12e7/DEPS#L398

This SDK was never meant to be published on beta and stable channels. Dart has recently changed the release tooling to no longer produce these SDKs on beta and stable. The same will apply to Linux RISCV SDKs but those are currently not used by Flutter.

Now Flutter can no longer receive beta and stable hotfixes from Dart because they are missing the Windows ARM64 SDKs. Dart 2.18.3 was not included in the latest Flutter hotfix because of this issue.

/cc @godofredoc @itsjustkevin @mit-mit

**Comment by mit-mit** [comment]
looks like this is from https://github.com/flutter/engine/pull/33681/files, cc @zanderso

**Comment by zanderso** [comment]
@cbracken is this safe to remove from Flutter master channel, or will it break folks working on/with Flutter Windows ARM64 support?

**Comment by cbracken** [comment]
From a separate chat this morning with @timsneath @godofredoc it sounds like this should be alright on master channel, but problematic to build on beta/stable channels. Is that the case?

It would be good to have the option to build for arm64 Windows on master until we've got to the point where we're able to properly support this with build/devicelab hardware.

**Comment by athomas** [comment]
> From a separate chat this morning with @timsneath @godofredoc it sounds like this should be alright on master channel, but problematic to build on beta/stable channels. Is that the case?
> 
> It would be good to have the option to build for arm64 Windows on master until we've got to the point where we're able to properly support this with build/devicelab hardware (and co-ordination with the Dart SDK side of things as well).

That's correct. Note that we might transition arm64 Windows to beta at some point before shipping it on stable.

**Comment by godofredoc** [comment]
closing this as complete. The arm64 artifacts are now built in their own build and running it or not on release branches is controlled from .ci.yaml file.
